starters/cms/src/App.css

```diff
@@ -178,3 +183,7 @@ form {
     background: linear-gradient(to bottom, transparent, var(--framer-color-bg));
     pointer-events: none;
 }
+
+.mapping footer > button {
+    text-transform: capitalize;
```

Comment:
Why this?

Reply:
It was there as `style` before, just moved to CSS, I suppose for collections that have lowercase words in them.